airbyte-server/src/main/java/io/airbyte/server/apis/DestinationOauthApiController.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.apis;
+
+import io.airbyte.api.generated.DestinationOauthApi;
+import io.airbyte.api.model.generated.CompleteDestinationOAuthRequest;
+import io.airbyte.api.model.generated.DestinationOauthConsentRequest;
+import io.airbyte.api.model.generated.OAuthConsentRead;
+import io.airbyte.api.model.generated.SetInstancewideDestinationOauthParamsRequestBody;
+import io.airbyte.server.handlers.OAuthHandler;
+import java.util.Map;
+import javax.ws.rs.Path;
+import lombok.AllArgsConstructor;
+
+@Path("/v1/destination_oauths")
```

Comment:
Double checking that `oauths` should be plural in the path.

Reply:
yep, it comes from the generated file:
```
@Path("/v1/source_oauths")
@Api(description = "the SourceOauth API")
@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaJAXRSSpecServerCodegen", date = "2022-11-02T15:07:06.363568-04:00[America/New_York]")
public interface SourceOauthApi {
```